tests/unit/AccountAuthorizationDialog.spec.ts

```diff
@@ -2,7 +2,7 @@ import Vue from 'vue'
 import Vuetify from 'vuetify'
 import { getVuexStore } from '@/store'
 import { shallowMount, mount } from '@vue/test-utils'
-import { AccountAuthorizationDialog } from '@/components/dialogs'
+import AccountAuthorizationDialog from '@/components/dialogs/AccountAuthorizationDialog.vue'
```

Comment:
Wherever possible, the unit tests should directly import the component(s) under test. This makes the unit tests more sensitive (fragile) to refactoring changes.

The only exception, because of how we export them, are the enums and interfaces. But even there I changed the tests to be as specific as possible.